extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/serde/uriExtractors.scala

```diff
@@ -33,3 +35,9 @@ class StringURIExtractor extends URIExtractor {
     Some(Uri(v1.asInstanceOf[String]))
   }
 }
+
+class CatalogStorageFormatURIExtractor extends URIExtractor {
+  override def apply(v1: AnyRef): Option[Uri] = {
```

Comment:
Are there any instances where a case visits multiple paths? If so, how about we change Option here to a Seq

Reply:
> Are there any instances where a case visits multiple paths? If so, how about we change Option here to a Seq

Nice suggestion, we have such cases such as HadoopFsRelation

Reply:
> Are there any instances where a case visits multiple paths? If so, how about we change Option here to a Seq

Done